tests/tracking/fluent/test_fluent.py

```diff
@@ -581,9 +587,9 @@ def test_start_run_with_parent():
         user_patch,
         source_name_patch,
     ):
-        active_run = start_run(experiment_id=mock_experiment_id, nested=True)
+        active_run = start_run(experiment_id=mock_experiment_id.return_value, nested=True)
```

Comment:
Since we're performing a type checking here now, we need to use the return_value of the Mock.